speechbrain/core.py

```diff
@@ -555,6 +582,14 @@ class and override any methods for which the default behavior does not
             The location for performing computations.
         precision (str)
             One of ``fp32``, ``fp16``, ``bf16``.
+        gradscaler_init_scale (float)
```

Comment:
I am not in favor in adding these parameters as basic Brain parameters. They will never be used by 99.99% of our users.